PhysicsTools/NanoAOD/plugins/SimpleFlatTableProducerPlugins.cc

```diff
@@ -3,6 +3,45 @@
 #include "DataFormats/Candidate/interface/Candidate.h"
 typedef SimpleFlatTableProducer<reco::Candidate> SimpleCandidateFlatTableProducer;
 
+#include "DataFormats/TrackReco/interface/Track.h"
```

Comment:
Given the large number of templates being instantiated here, it might help compilation time and memory used by the compiler to split these instantiations into multiple files.

Reply:
@hqucms ping

Reply:
Done in https://github.com/cms-sw/cmssw/pull/44782/commits/782b7344502b2b698b8032ded2f00481ec39ceee.